framework/sandstone_p.h

```diff
@@ -443,11 +447,13 @@ struct SandstoneApplication::SharedMemory
     int child_debug_socket = -1;
 #endif
 
-    LogicalProcessorSet enabled_cpus;
+    int total_cpu_count = 0;
+    alignas(64) struct cpu_info cpu_info[];         // C99 Flexible Array Member
 
-    // in/out per-thread data
-    PerThreadData::Main main_thread_data;
-    PerThreadData::Test per_thread[MAX_THREADS];
+    // in/out per-thread data allocated dynamically
+//  alignas(PAGE_SIZE)
```

Comment:
Please remove.

Reply:
It was intentional. It's not code I commented out, it's a comment to indicate that this content is aligned.

I'll rephrase.